### PR TITLE
add derived features nb

### PR DESCRIPTION
**Pull request by slemasne**

### CODE REVIEW

File: examples/features/core-lusid/Derived portfolios.ipynb [hunk 1]
```diff
@@ -0,0 +1,898 @@
+{
```
Comment (rizwansaeed) [inline examples/features/core-lusid/Derived portfolios.ipynb:1]: <p>I wouldn't say "copied", perhaps "A derived portfolio is a portfolio which inherits the contents (or "derived") from another parent portfolio"</p><p> </p><p>They are not limited to transactions (afaik)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (rizwansaeed) [inline examples/features/core-lusid/Derived portfolios.ipynb:1]: <p>maybe a stylistic preference, but I tend to put the descriptive text into a markdown line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (rizwansaeed) [inline examples/features/core-lusid/Derived portfolios.ipynb:1]: <p>apparently its better to have the scope constant and add the guid to the code (from a performance perspective)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (rizwansaeed) [inline examples/features/core-lusid/Derived portfolios.ipynb:1]: <p>section </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (slemasne) [reply examples/features/core-lusid/Derived portfolios.ipynb:1]: <p>Sure, done. </p>

Comment (slemasne) [reply examples/features/core-lusid/Derived portfolios.ipynb:1]: <p>Sure. Done. Consistency is important. </p>

Comment (slemasne) [reply examples/features/core-lusid/Derived portfolios.ipynb:1]: <p>I saw the comments in slack earlier.</p><p>That makes sense.</p>

Comment (slemasne) [reply examples/features/core-lusid/Derived portfolios.ipynb:1]: <p>I have updated the code. </p>

Comment (slemasne) [reply examples/features/core-lusid/Derived portfolios.ipynb:1]: <p>Sure, done. </p>

**Pull request merged**